src/database/users.ts

```diff
@@ -22,25 +24,204 @@ export async function deleteUser(userId: string) {
 }
 
 /** Add new user to the database if they do not exist already */
-export async function createNewUser(userId: string, guildId: string, coins?: number) {
-    await prisma.user.upsert({
+export function createNewUser(userId: string) {
+    return prisma.user.upsert({
         where: {
             id: userId
         },
         update: {},
         create: {
             id: userId,
+        },
+    });
+}
+
+//#MARKER members
+
+/** Gets member via ID */
+export function getMember(guildId: string, userId: string) {
+    return prisma.member.findUnique({
+        where: {
+            guildId_userId: {
+                guildId,
+                userId,
+            },
+        },
+    });
+}
+
+/** Remove member from the database */
+export async function deleteMember(guildId: string, userId: string) {
+    await prisma.member.delete({
+        where: {
+            guildId_userId: {
+                guildId,
+                userId,
+            },
+        },
+    }).catch(); // this will be updated once prisma implements a doesNotExist thing
```

Comment:
you can do something like i did in `database/contest.ts` line 276.

Reply:
That line was from canny I think

Reply:
fair enough lol

Reply:
copied it over tho